models/google_ads.yml

```diff
@@ -312,9 +312,9 @@ models:
             - source_relation
             - ad_group_id
             - search_term
-            - keyword_text
```

Comment:
Should we still include keyword_text in addition to the criterion_id?

Reply:
We don't use `keyword_text` for the keyword_report test (we use `criterion_id` instead), so I lean toward sticking with `criterion_id` as the true keyword identifier. Open to other opinions tho